pkgs/by-name/po/portmaster/package.nix

```diff
@@ -0,0 +1,55 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+, iptables
+}:
+
+buildGoModule rec {
+  pname = "portmaster";
+  version = "1.6.0";
+  CGO_ENABLED = 0;
+
+  ldflags =
+    let
+      BUILD_PATH = "github.com/safing/portbase/info";
+      BUILD_COMMIT = "v${version}";
+      BUILD_USER = "nixpkgs";
+      BUILD_HOST = "hydra";
+      BUILD_DATE = "31.10.2023";
+      BUILD_SOURCE = src.gitRepoUrl;
+      BUILD_BUILDOPTIONS = "";
```

Comment:
This `let ... in` is not really needed - you can simply write the `ldflags` as is.

Reply:
Additionally, usually derivation variables such as this `ldflags` array is set after the `src` and `version`. This makes the update diffs smaller, since the hash and the version string changes are close to each other.